src/utils/helpers.py

```diff
@@ -2,6 +2,26 @@
 from pathlib import Path
 
 
+def clear_directory(dir: Path | str) -> None:
+    """
+    Clears all files and subdirectories in the specified directory.
+
+    Args:
+        directory_path: A string or Path object to the target directory,
+            relative to the Docker container.
+    """
+    dir_path = Path(dir)
+    if not dir_path.is_dir():
+        raise ValueError(f"The path {dir} is not a valid directory.")
+
+    for item in dir_path.iterdir():
+        if item.is_dir():
+            clear_directory(item)  # Recursively clear
+            item.rmdir()
```

Comment:
Yeah, me being dumb again. Fixed in 3e82cca.

Reply:
Nice! This may be intentional, but heads up that (by my reading) the old version of `clear_directory` would preserve the existence of the empty top-level directory itself, while `shutil.rmtree` will delete everything, including the empty directory. I'm not sure if that matters, so disregard if not!

Reply:
Tis fine, as the top level dir is re-created on every Spark read.